.travis.yml

```
@@ -1,6 +1,8 @@
 language: ruby
 
-sudo: false
+sudo: required
+addons:
+  chrome: stable
```

```suggestion
  apt:
    packages:
      - chromium-chromedriver
```
https://github.com/travis-ci/travis-ci/issues/272#issuecomment-389598926

According to that, you also have to add: 
```
before_script:
  - ln -s /usr/lib/chromium-browser/chromedriver ~/bin/chromedriver
```

Hopefully that works, I haven't tried it out personally.

---

@ta1kt0me Do you think you can try?

Thanks @BobbyMcWho for the suggestion

---

@BobbyMcWho @benoittgt 
Thank you for advice. I'll try it.